.github/workflows/nodejs.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
```

Comment:
I believe we're only doing automated testing on LTS versions of Node.js.

@delvedor can you confirm that's the case? Docs aren't totally clear on what non-EOL versions of Node we support, and I don't want the code to imply support for a version that we don't support.